src/web/css/shared.css

```diff
@@ -14,7 +16,7 @@ em {
   width: 50%;
   height: 100%;
   z-index: 8700;
-  background: url('/img/pyret-spin.gif') 50% 30% no-repeat rgba(249, 249, 249, 0.5);
+  background: url('/img/patch-treadmill-run.gif') 50% 30% no-repeat rgba(249, 249, 249, 0.5);
```

Comment:
Does this affect CPO as well as Patch?

Reply:
No. shared.css is used only by Pyret (via editor.html). Its sister file, shared-patch.css, is used only by Patch (via editor-patch.html). When running Patch, editor-patch.html is copied to editor.html. For nomenclatural consistency, I've added editor-pyret.html, which is exactly the original editor.html.

A clone of the code.pyret.org repo will run Pyret by default. In order to have it run Patch, copy editor-patch.html to editor.html, and have the pyret-mode.js line in the Makefile point to patch.js rather than pyret.js. This is two changes, and it would of course be nice if it could be reduced into just one change, presumably just the Makefile where the 'cp editor-patch.html editor.html' can be inserted. My Makefile skills are a bit rusty: Maybe a dialect argument to the Makefile can avoid the need for having to modify the Makefile at all.

Reply:
I see what you mean. The changes to shared.css in the patch-a11y branch should not be merged. (They are not used by Patch anyway for a long time, all the Patch-specific changes having been consolidated into the new file shared-patch.css, but I neglected to undo my changes to shared.css.)

Reply:
Right.  This is the `shared.css` file, not the `shared-patch.css` file.